tests/integrationtest/r/imdbload.result

```diff
@@ -277,95 +277,95 @@ load stats 's/imdbload_stats/movie_info.json';
 load stats 's/imdbload_stats/cast_info.json';
 explain select * from char_name where ((imdb_index = 'I') and (surname_pcode < 'E436')) or ((imdb_index = 'L') and (surname_pcode < 'E436'));
 id	estRows	task	access object	operator info
-IndexLookUp_10	1005030.94	root		
-├─IndexRangeScan_8(Build)	1005030.94	cop[tikv]	table:char_name, index:itest2(imdb_index, surname_pcode, name_pcode_nf)	range:["I" -inf,"I" "E436"), ["L" -inf,"L" "E436"), keep order:false
-└─TableRowIDScan_9(Probe)	1005030.94	cop[tikv]	table:char_name	keep order:false
+IndexLookUp_10	1004193.52	root		
```

Comment:
The missing should be expected since they are not used as predicates, but the `imdb_index:unInitialized` might be unexpected. I need to take a look.